src/build_helper/lib.rs

```diff
@@ -279,8 +277,11 @@ pub fn sanitizer_lib_boilerplate(sanitizer_name: &str)
     } else {
         format!("static={}", link_name)
     };
+    // The source for `compiler-rt` comes from the `compiler-builtins` crate, so
+    // load our env var set by cargo to find the source code.
+    let dir = env::var_os("DEP_COMPILER_RT_COMPILER_RT").unwrap();
```

Comment:
Is this a new feature of Cargo? I didn't know this was possible!

Reply:
Nah this has been in there since 1.0 :)